### PR TITLE
Serialize tests [Jira: DCK-79]

### PR DESCRIPTION
**Pull request by SamHellawell**
Fixes https://github.com/docknetwork/sdk/issues/73

### CODE REVIEW

File: example/resolver.js [hunk 1]
```diff
@@ -78,24 +78,22 @@ async function main() {
 
   console.log('Building DIDs list...');
 
+  const dockDID = await createDockDID();
```
Comment (SamHellawell) [inline example/resolver.js:81]: had to make a side change here, example was hanging in the CI process. some dids were duplicates and/or not resolving properly, causing a hang but not an error. something in universal resolver side

File: src/modules/schema.js [hunk 2]
```diff
@@ -27,6 +27,24 @@ export default class Schema {
     this.id = id || createNewDockBlobId();
   }
 
+  static fromJSON(json) {
+    const {
+      id, schema, author, ...rest
```
Comment (lovesh) [inline src/modules/schema.js:32]: The only extra property is `signature` here, correct? Would we want a schema object with additional properties? If not then we should only check for the ones we know about and skip others.

File: src/verifiable-presentation.js [hunk 3]
```diff
@@ -39,6 +39,42 @@ class VerifiablePresentation {
...
+        vp.addType(type);
+      }
+    }
+
+    const context = rest['@context'];
+    if (context) {
+      vp.context = rest['@context'];
+      delete rest['@context'];
+    }
+
```
Comment (lovesh) [inline src/verifiable-presentation.js:73]: Not having a context should raise an error as it won't be a valid JSON-LD object.

File: src/modules/schema.js (hunk 2 above)
Comment (SamHellawell) [reply src/modules/schema.js]: yeah and we omit signature in our tojson method. good point here, have updated to remove need for rest

File: src/verifiable-presentation.js (hunk 3 above)
Comment (SamHellawell) [reply src/verifiable-presentation.js]: good point, resolved

**Pull request merged**